microsoft-365/enterprise/cross-tenant-mailbox-migration.md

```diff
@@ -406,7 +398,11 @@ While Teams meetings are moved, the meeting URL isn't updated when items migrate
 
 ### What content is migrated cross-tenant?
 
-When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions, and Purges are migrated.  
+When a mailbox is migrated cross-tenant with this feature, only user-visible content in the mailbox, also known as Top of Information Store (email, contacts, calendar, tasks, and notes), and the Recoverable Items folders Deletions, Versions, and Purges are migrated.
+
+### Do items in the Outbox get migrated cross-tenant?
+
+Items in the Outbox are not migrated cross-tenant as this folder is a client based folder specific to the Outlook client. Items in the Outbox are stored locally, and not synced to the cloud.
```

Comment:
```suggestion
Items in the Outbox are not migrated cross-tenant as this folder is a client-based folder specific to the Outlook client. Items in the Outbox are stored locally, and not synced to the cloud.
```

Reply:
@JohanFreelancer9 This has been changed and committed.